mifosng-android/build.gradle

```diff
@@ -256,6 +264,8 @@ dependencies {
     implementation 'androidx.cardview:cardview:1.0.0'
     //preferences
     implementation "androidx.preference:preference:$preference"
+
+    implementation 'com.github.danishjamal104:mifos-android-sdk-arch:2.1.1-alpha'
```

Comment:
This will be changed to `com.github.openMF:mifos-android-sdk-arch:2.0.0-alpha`, soon after merging [mifos-android-sdk-arch#15](https://github.com/openMF/mifos-android-sdk-arch/pull/15)